src/fluentexpr.prw

```diff
@@ -18,10 +18,20 @@ Static Function Format( cString, aValues )
     Local cResult := cString
     Local nIndex
     For nIndex := 1 To Len( aValues )
-        cResult := StrTran( cResult, '{' + AllTrim( Str( nIndex ) ) + '}', cValToChar( aValues[ nIndex ] ) )
+        cResult := StrTran( cResult, '{' + AllTrim( Str( nIndex ) ) + '}', ToString( aValues[ nIndex ] ) )
     Next
     Return cResult
 
+Static Function ToString( xValue )
+    If ValType( xValue ) == "A"
+        Return '{ ' + ArrTokStr(xValue, ', ') + ' }'
```

Comment:
The operation for arrays must be recursive, otherwise subarrays will not be rendered.
```xbase
'{ ' + ArrTokStr( AEval( xValue, { |xExpr| ToString( xExpr ) } ), ', ' ) + ' }'
```
 should work

Reply:
`ArrTokStr` already return all values as `string`, you can see at the first image of my PR.